src/utils/crypto.utils.ts

```diff
@@ -0,0 +1,10 @@
+import {uint8ArrayToHexString} from '@dfinity/utils';
+
+export async function generateHash(params: Record<string, string>): Promise<string> {
```

Comment:
To improve types, do we have in the code a Zod schema for Hex or Base64 string? If so, what do you think of using that types of result?